rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto

```diff
@@ -1041,10 +1041,11 @@ message NervousSystemParameters {
   // This parameter determines the maximum time period that the voting period
   // may be extended after a flip. If there is a flip at the very end of the
   // original proposal deadline, the remaining time will be set to this parameter.
-  // If there is a flip before or after the original deadline, the deadline will
+  // If there is a flip before or after the original deadline, the deadline will be
   // extended by somewhat less than this parameter.
   // The maximum total voting period extension is 2 * wait_for_quiet_deadline_increase_seconds.
   // For more information, see the wiki page on the wait-for-quiet algorithm:
+  // Shah-TODO link doesn't exist
```

Comment:
The Wiki is now deprecated, maybe we could refer to a page on https://internetcomputer.org/ instead
```suggestion
  // Shah-TODO link doesn't exist
```